src/containers/pages/IRS/Map/index.tsx

```diff
@@ -0,0 +1,332 @@
+  SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICES,
+  SUPERSET_IRS_REPORTING_PLANS_SLICE,
+  SUPERSET_IRS_REPORTING_STRUCTURES_DATA_SLICE,
+  SUPERSET_JURISDICTIONS_SLICE,
+} from '../../../../configs/env';
+import {
+  indicatorThresholdsIRS,
+  sidebarIndicatorRowsIRS,
+  sidebarLegendStopsIRS,
+} from '../../../../configs/settings';
+import { HOME, HOME_URL, IRS_REPORTING_TITLE, REPORT_IRS_PLAN_URL } from '../../../../constants';
+import ProgressBar from '../../../../helpers/ProgressBar';
+import { RouteParams } from '../../../../helpers/utils';
+import supersetFetch from '../../../../services/superset';
+import GenericJurisdictionsReducer, {
+  fetchGenericJurisdictions,
+  GenericJurisdiction,
+  getGenericJurisdictionByJurisdictionId,
+  reducerName as GenericJurisdictionsReducerName,
+} from '../../../../store/ducks/generic/jurisdictions';
+import IRSPlansReducer, {
+  fetchIRSPlans,
+  getIRSPlanById,
+  IRSPlan,
+  reducerName as IRSPlansReducerName,
+} from '../../../../store/ducks/generic/plans';
+import genericStructuresReducer, {
+  fetchGenericStructures,
+  GenericStructure,
+  getGenericStructures,
+  reducerName as genericStructuresReducerName,
+  StructureFeatureCollection,
+} from '../../../../store/ducks/generic/structures';
+import jurisdictionReducer, {
+  fetchJurisdictions,
+  getJurisdictionById,
+  Jurisdiction,
+  reducerName as jurisdictionReducerName,
+} from '../../../../store/ducks/jurisdictions';
+import { getGisidaWrapperProps, getIndicatorRows, getJurisdictionBreadcrumbs } from './helpers';
+import './style.css';
+
+/** register the reducers */
+reducerRegistry.register(IRSPlansReducerName, IRSPlansReducer);
+reducerRegistry.register(jurisdictionReducerName, jurisdictionReducer);
+reducerRegistry.register(GenericJurisdictionsReducerName, GenericJurisdictionsReducer);
+reducerRegistry.register(genericStructuresReducerName, genericStructuresReducer);
+
+const slices = SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICES.split(',');
+const focusAreaSlice = slices.pop();
+
+/** interface for IRSReportingMap */
+interface IRSReportingMapProps {
+  fetchFocusAreas: typeof fetchGenericJurisdictions;
+  fetchJurisdictionsAction: typeof fetchJurisdictions;
+  fetchPlans: typeof fetchIRSPlans;
+  fetchStructures: typeof fetchGenericStructures;
+  focusArea: GenericJurisdiction | null;
+  jurisdiction: Jurisdiction | null;
+  plan: IRSPlan | null;
+  service: typeof supersetFetch;
+  structures: StructureFeatureCollection | null;
+}
+
+/** IRSReportingMap default props */
+const defaultProps: IRSReportingMapProps = {
+  fetchFocusAreas: fetchGenericJurisdictions,
+  fetchJurisdictionsAction: fetchJurisdictions,
+  fetchPlans: fetchIRSPlans,
+  fetchStructures: fetchGenericStructures,
+  focusArea: null,
+  jurisdiction: null,
+  plan: null,
+  service: supersetFetch,
+  structures: null,
+};
+
+/** The IRS Reporting Map component */
+const IRSReportingMap = (props: IRSReportingMapProps & RouteComponentProps<RouteParams>) => {
```

Comment:
the new IRS reporting map component

Reply:
@moshthepitt this is gorgeous, good work man